core/jazz_ui/src/app/secondary-components/create-service/oss/create-service.component.html

```diff
@@ -80,20 +80,20 @@ <h4 class="bold-title">
                         </section>
                         <section class="col-lg-3 col-md-3 pad-left0">
                             <div class="radio-container">
-                                <input type="radio" name="runtime" id="java" [value]="java" [checked]="runtime == 'java'" (click)="onSelectionChange('java')">
-                                <label for="java"><span class='outer'></span><span class="background"></span>Java 8</label>
+                                <input type="radio" name="runtime" id="python2-7" [value]="python2-7" [checked]="runtime == 'python2.7'" (change)="onSelectionChange('python2.7')">
+                                <label for="python2-7"><span class='outer'></span><span class="background"></span>Python (2.7)</label>
```

Comment:
Suggesting these changes - 
Python (2.7) => Python 2.7
python2-7 => python2.7

Reply:
Done for Python (2.7) => Python 2.7